content/en/references/docker-images.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Docker images
+weight: 100
+tags:
+- docker
+description: >
+  Overview of LocalStack Docker images and their usage
+---
+
+LocalStack functions as a local “mini-cloud” operating system that runs inside a Docker container. LocalStack has multiple components, which include process management, file system abstraction, event processing, schedulers, and more. Running inside a Docker container, LocalStack exposes external network ports for integrations, SDKs, or CLI interfaces to connect to LocalStack APIs. The LocalStack & LocalStack Pro Docker images have been downloaded over 130+ million times and provide a multi-arch build compatible with AMD/x86 and ARM-based CPU architectures. This section will cover the different Docker images available for LocalStack and how to use them.
+
+## LocalStack Community image
+
+The LocalStack Community image (`localstack/localstack`) contains the community and open-source version of our [core cloud emulator](https://github.com/localstack/localstack). To use the LocalStack Community image, you can pull the image from Docker Hub:
+
+{{< command >}}
+$ docker pull localstack/localstack:latest
+{{< / command >}}
+
+To use the LocalStack Community image, you don't need to sign-up for an account on [LocalStack Web Application](https://app.localstack.cloud). The Community image is free to use and does not require any API key to run. The Community image can be used to run [local AWS services](https://docs.localstack.cloud/user-guide/aws/) with [integrations](https://docs.localstack.cloud/user-guide/integrations/) on your local machine or in your [continuous integration pipelines](https://docs.localstack.cloud/user-guide/ci/). 
+
+The Community image also covers a limited set of [LocalStack Tools](https://docs.localstack.cloud/user-guide/tools/) to make your life as a cloud developer easier. However, the [LocalStack Web Application](https://docs.localstack.cloud/user-guide/web-application/) is not supported! You can use [LocalStack Cockpit](https://localstack.cloud/products/cockpit/) or [LocalStack Docker Extension](https://docs.localstack.cloud/user-guide/tools/localstack-docker-extension/) to use LocalStack with a graphical user interface.
```

Comment:
I'd rather not mention that the Web app is not supported in Community, but rather highlight in the section below that Pro comes with the full Web experience. (which is already the case in the Pro section)
```suggestion
The Community image also covers a limited set of [LocalStack Tools](https://docs.localstack.cloud/user-guide/tools/) to make your life as a cloud developer easier. You can use [LocalStack Cockpit](https://localstack.cloud/products/cockpit/) or [LocalStack Docker Extension](https://docs.localstack.cloud/user-guide/tools/localstack-docker-extension/) to use LocalStack with a graphical user interface.
```